Source/Core/DolphinWX/ISOProperties/FilesystemPanel.cpp

```diff
@@ -68,20 +68,39 @@ wxImageList* LoadIconBitmaps(const wxWindow* context)
   return icon_list;
 }
 
+// Iterates over all files in a directory summing up their sizes to determine the size of the directory
+u64 DetermineDirectorySize(const DiscIO::FileInfo& directory, u64 size)
```

Comment:
Also, it seems like the `size` argument always is 0 in practice. You should remove that parameter.